pkg/contexts/ocm/accessmethods/npm/method.go

```diff
@@ -120,8 +116,6 @@ func (a *AccessSpec) getPackageMeta(ctx accspeccpi.Context) (*meta, error) {
 	return &metadata, nil
 }
 
-////////////////////////////////////////////////////////////////////////////////
-
```

Comment:
this comment separates two object types, spec and method and should be kept.

Reply:
aha! ok